data_centre/data.py

```diff
@@ -33,6 +51,7 @@ def __init__(self, message_handler):
         #self.EMPTY_BANK = [self.EMPTY_SLOT for i in range(10)]
         self.PATHS_TO_BROWSER = [self.PATH_TO_EXTERNAL_DEVICES, '/home/pi/Videos' ]
         self.PATHS_TO_SHADERS = [self.PATH_TO_EXTERNAL_DEVICES, '/home/pi/r_e_c_u_r/Shaders', '/home/pi/Shaders' ]
+        self.PATHS_TO_PLUGIN_DATA = [ '/home/pi/r_e_c_u_r/json_objects/plugins', self.PATH_TO_EXTERNAL_DEVICES ]
```

Comment:
i dont think this is correct - why would there be plugin data in the external device ?

Reply:
Ah yes probably not - might've been thinking that plugins could be .py files stored on external device too, for making it easier to edit and add plugins (download and put on thumb drive). Haven't tested if that works though